### ISSUE TITLE
[Feature Request]: n8n integration

### ISSUE THREAD

**Issue by tiger360** [carboneio/carbone#146]
Hey

It would be very interesting to integrate carbonejs to n8n.
It's a  Workflow Automation Tool.

Thanks

**Comment by osulluke** [comment]
Yes, having an embedded open source PDF library would be a very helpful addition to n8n.
